libs/datamodel/core/tests/types/postgres_native_types.rs

```diff
@@ -1,21 +1,21 @@
 use crate::common::*;
-use crate::types::helper::{test_native_types_with_field_attribute_support, test_native_types_without_attributes};
+use crate::types::helper::test_native_types_without_attributes;
 use datamodel::{ast, diagnostics::DatamodelError};
 use native_types::PostgresType;
 
-#[test]
-fn should_fail_on_serial_data_types_with_number_default() {
-    fn error_msg(type_name: &str) -> String {
-        format!(
-            "Sequential native type {} of Postgres must not have a static default value.",
-            type_name
-        )
-    }
-
-    for tpe in &["SmallSerial", "Serial", "BigSerial"] {
-        test_native_types_with_field_attribute_support(tpe, "Int", "default(4)", &error_msg(tpe), POSTGRES_SOURCE);
-    }
-}
+// #[test] no more serial currently
+// fn should_fail_on_serial_data_types_with_number_default() {
+//     fn error_msg(type_name: &str) -> String {
+//         format!(
+//             "Sequential native type {} of Postgres must not have a static default value.",
+//             type_name
+//         )
+//     }
+//
+//     for tpe in &["SmallSerial", "Serial", "BigSerial"] {
+//         test_native_types_with_field_attribute_support(tpe, "Int", "default(4)", &error_msg(tpe), POSTGRES_SOURCE);
+//     }
+// }
```

Comment:
Do you think we can delete this, or should we leave it in until we've had the opportunity to discuss serial types with product?

Reply:
Yeah, wanted to leave it in due to that reason